CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Features
 - Add experimental runtime accessibility attributes validation (the initial step validates the Button component only) @mshoho ([#1911](https://github.com/stardust-ui/react/pull/1911))
 
+### Features
+- Export `bot`, `tab-badge` and `connector-badge` icon to Teams theme @cloudtx ([#1968](https://github.com/stardust-ui/react/pull/1968))
```

Comment:
NIT: Why do we add `-badge` to the `tab` and `connector` icon names but not to the `bot` one? How are they different?

Reply:
I think they pulled these icons over from the angular app where they had the same naming. We could change them if need be.

Reply:
agree - please rename to things that are generic to the image, not the use case. Like fontawesome, I think the first one should be name `robot`. The other 2 perhaps `tabs` and `plugs`

Reply:
icon names changed